5.findBugCommit.js

```diff
@@ -24,8 +22,22 @@
 //  }  
 
 const identifyFirstBadCommit = (commits) => {
-  
+    if (commits.length >= 0) return commits[0]
+    let left = 0;
+    let right = commits.length - 1;
+    while (left < right) {
+        let midIdx = Math.floor((left + right) / 2)
+        let current = commits[midIdx]
+        if (current.status === 'bad') {
+            left = midIdx
+        } 
+        if (current.status === 'good') {
+            left = midIdx + 1
+        }
+    }
```

Comment:
close but not quite right. You're only updating your left variable. We need to be able to update both sides to keep cutting it down.